docs/advanced-features/instrumentation.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Learn how instrument your Next.js app.
+---
+
+> **Note**: This feature is experimental, you need to explicitly opt-in by providing `experimental.instrumentationHook = true;` in your `next.config.js`.
+
+# `instrumentation.ts`
+
+You can export `register` function from this file and we will be called whenever we bootstrap a new server instance.
```

Comment:
```suggestion
If you export a function named `register` from this file, Vercel will be call that function whenever we bootstrap a new server instance.
```

Feel free to ignore this suggestion if I'm diverting away from what you originally meant